projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/methods-and-properties.migration.ts

```diff
@@ -688,13 +937,104 @@ export const METHODS_AND_PROPERTIES_DEPRECATIONS_DATA: MethodPropertyDeprecation
       comment:
         "// TODO:Spartacus - Property 'activeCartService' was removed from Class 'CartVoucherService'. ",
     },
+    {
+      class: 'CartVoucherService',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'store',
+      comment:
+        "The type of property 'store' changed to: 'store: Store<StateWithProcess<void>>' ",
+    },
+    {
+      class: 'Command',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'execute',
+      comment:
+        "The 'execute' method's signature changed to: 'execute( parameters: PARAMS): Observable<RESULT>'",
```

Comment:
These cases may or may not result in breaking changes.  In this specific occurrence, it might result in customer custom code having compilation errors.